### PR TITLE
[Documentation][Cookbook] How to add another type of refund?

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | 1.9
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes https://github.com/Sylius/RefundPlugin/issues/279 and waits for https://github.com/Sylius/RefundPlugin/pull/323
| License         | MIT

screencapture-localhost-63342-Sylius-docs-build-cookbook-payments-custom-type-of-refund-html-2021-07-06-11_37_48

### CODE REVIEW

File: docs/cookbook/payments/custom-type-of-refund.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+.. note::
+
+    This cookbook describes customization of a feature available only with `Sylius/RefundPlugin <https://github.com/Sylius/RefundPlugin/>`_ installed.
+
+Why would you add type of refund?
+---------------------------------
+
+Refund Plugin provides a generic solution for refunding orders, it is enough for a basic refund but many shops need more custom functionalities.
+For example, one may need to add loyalty points as a different refund type than order item unit and shipment.
```
Comment (AdamKasp) [inline docs/cookbook/payments/custom-type-of-refund.rst:12]:
```suggestion
For example, one may need to add loyalty points (Sylius Plus feature) as a different refund type than order item unit and shipment.
```
wdyt?

**Review by AdamKasp — APPROVED** [review]

File: docs/cookbook/payments/custom-type-of-refund.rst (hunk 1 above)
Comment (GSadee) [reply docs/cookbook/payments/custom-type-of-refund.rst:12]: @CoderMaggie, wdyt?

File: docs/cookbook/payments/custom-type-of-refund.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        {
+            return new self(self::LOYALTY);
+        }
+    }
+
+You need also to set the parameter with new RefundType in your configuration file:
+
+.. code-block:: yaml
+
+    # config/packages/sylius_refund.yaml.yaml
```
Comment (arti0090) [inline docs/cookbook/payments/custom-type-of-refund.rst:68]:
```suggestion
    # config/packages/sylius_refund.yaml
```

Is the .yaml yamled? :exploding_head:

**Review by SirDomin — APPROVED** [review]

**Review by arti0090 — APPROVED** [review]

**Comment by AdamKasp** [comment]
Thank you, Grzegorz! :tada:

**Pull request merged**